script.js

```diff
@@ -0,0 +1,30 @@
+function randn_bm() { 
+  let u = 0, v = 0; 
+  while (u === 0) u = Math.random(); 
+  while (v === 0) v = Math.random(); 
+  return Math.sqrt(-2.0 * Math.log(u)) * Math.cos(2.0 * Math.PI * v) * 2 | 0; 
+};
+
+function createGaussObject(num) {
+  let obj = {};
```

Comment:
тут лучше const, let лучше там где нужно перезаписывать переменную
```suggestion
  const obj = {};
```

Reply:
Done!